app/src/main/java/jp/co/soramitsu/app/root/navigation/Navigator.kt

```diff
@@ -598,6 +600,20 @@ class Navigator :
     }
 
     override fun openEducationalStories(stories: StoryGroupModel) {
-        navController?.navigate(R.id.open_staking_story, StoryFragment.getBundle(stories))
+        navController?.navigate(R.id.onboardingStoriesFragment, StoryFragment.getBundle(stories))
     }
+
+    fun educationalStoriesCompleted() {
+        navController!!.previousBackStackEntry!!.savedStateHandle.set(StoryFragment.KEY_STORY, true)
```

Comment:
change everywhere possible "!!" to "?"; we better not crash if somehow user got null here